alica_engine/src/engine/Assignment.cpp

```diff
@@ -117,14 +117,16 @@ bool Assignment::isSuccessful() const
     }
     const EntryPointGrp& eps = _plan->getEntryPoints();
     const int numEps = eps.size();
+    bool ret = false;
     for (int i = 0; i < numEps; ++i) {
         if (eps[i]->isSuccessRequired()) {
-            if (_successData.getRaw()[i].empty() || static_cast<int>(_successData.getRaw()[i].size()) < eps[i]->getMinCardinality()) {
+            if (!(_successData.getRaw()[i].empty() && static_cast<int>(_successData.getRaw()[i].size()) >= eps[i]->getMinCardinality())) {
```

Comment:
This is now incorrect. If _successData.getRaw()[i] is not empty, then isSuccessful will always return false.

Reply:
good find. Seems to be a bad merge